sphere_scene.py

```diff
@@ -140,13 +140,28 @@ class BouncingSphereScene(Scene):
 
     RENDER_FADE_MARGIN = 0.2  # Fade out near the edge
 
-    def __init__(self):
+    def __init__(self, config: dict):
+        """
+        Initializes the scene using a configuration dictionary.
+
+        Args:
+            config (dict): A dictionary containing scene settings, including
+                           the total width, height, and length of the
+                           volumetric display space.
+        """
         self.spheres: List[Sphere] = []
         self.next_spawn = 0.0
         self.spawn_interval = 2.0  # New sphere every 2 seconds
 
-    def spawn_sphere(self, time: float, bounds: tuple[float, float, float]) -> Sphere:
-        width, height, length = bounds
+        # Read the total dimensions of the scene from the config
+        self.width = config.get("width", 32)
+        self.height = config.get("height", 32)
+        self.length = config.get("length", 32)
+        self.bounds = (self.width, self.height, self.length)
```

Comment:
I think the raster object has the dimensions, but it only gets passed around in the render loop. It would probably be better to tell each scene what the dmensions are at init time like this, but I would prefer if it's in a structured data format. Maybe have a `DisplayProperties` dataclass that gets constructed from the config in `sender.py` and provided at init to each scene?